sites/svelte-5-preview/src/routes/docs/content/01-api/02-runes.md

```diff
@@ -112,6 +112,25 @@ Svelte provides reactive `Map`, `Set` and `Date` classes. These can be imported
 <p>{map.get('message')}</p>
 ```
 
+## `$state.clean`
+
+To remove reactivity from objects and arrays created with `$state`, use `$state.clean`:
+
+```svelte
+<script>
+	let counter = $state({ count: 0 });
+
+	$effect(() => {
+		// Will log { count: 0 }
+		console.log($state.clean(counter));
+	});
+</script>
+```
+
+This is handy when you want to pass some state to an external library or API that doesn't expect a reactive object – such as `structuredClone`.
```

Comment:
I know this matches the description we had before - but do we want to mention something specifically about it being the `Proxy` that causes the issue? In most typical cases, I think the fact that the object is reactive should be transparent to third-party code, correct?

Reply:
agree with this — have tweaked the docs